pkg/manifests/config.go

```diff
@@ -215,14 +217,19 @@ func NewConfig(content io.Reader, tp bool) (*Config, error) {
 	// The operator should only create some manifests if techPreview is enabled
 	c.TechPreview = tp
 
+	cmometrics.ConfiguredCollectionProfile.WithLabelValues("").Set(0)
```

Comment:
Iiuc this is a no-op, since `CollectionProfile` will default to `full`: https://github.com/openshift/cluster-monitoring-operator/blob/ec2e0d3da7199f68c73f5a081ebf2a131d56f243/pkg/manifests/config.go#L303

Reply:
Ah, right!